client/index.html

```diff
@@ -146,13 +146,13 @@
         <table id="mediathek" class="table table-striped table-hover">
           <thead>
             <tr>
-              <th>Sender</th>
-              <th>Thema</th>
-              <th>Titel</th>
+              <th data-onclick-sort="channel">Sender</th>
+              <th fieldata-is-disabled="topic">Thema</th>
+              <th fieldata-is-disabled="title">Titel</th>
```

Comment:
sure "fieldata" and not "fielddata"?

Reply:
Thanks for spotting the typo. There was no functional deficit, though...